src/test/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://${AQTS_DATABASE_ADDRESS}:${AQTS_DATABASE_PORT}/${AQTS_DATABASE_NAME}?currentSchema=${AQTS_SCHEMA_NAME}
```

Comment:
@dsteinich is this the offending `currentSchema` you mentioned? Probably in the src/main/resources application.yml as well...

Reply:
I'll leave this for now, I notice we have it in the other SB lambdas as well.